netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -180,6 +181,18 @@ public NettyServerBuilder sslContext(SslContext sslContext) {
   }
 
   /**
+   * Sets the {@link ProtocolNegotiator} to be used. If non-{@code null}, overrides the value
+   * specified in {@link #sslContext(SslContext)}.
+   *
+   * <p>Default: {@code null}.
+   */
+  public final NettyServerBuilder protocolNegotiator(
```

Comment:
The entire class is marked as `@ExperimentalApi`.  I suspect that should be sufficient.